packages/react-three-rapier/src/types.ts

```diff
@@ -169,8 +169,44 @@ export interface UseColliderOptions<ColliderArgs> {
    * The rotation of this collider relative to the rigid body
    */
   rotation?: Vector3Array;
+
+  /**
+   * Callback when this collider collides with another collider.
+   */
+  onCollisionEnter?: CollisionEnterCallback;
+
+  /**
+   * Callback when this collider stops colliding with another collider.
+   */
+  onCollisionExit?: CollisionExitCallback;
+
+  /**
+   * The bit mask configuring the groups and mask for collision handling.
+   */
+  collisionGroups?: InteractionGroups;
+
+  /**
+   * The bit mask configuring the groups and mask for solver handling.
+   */
+  solverGroups?: InteractionGroups;
+}
+
+export type CollisionEnterPayload = {
+  target: RapierRigidBody;
+  collider: RapierCollider;
+  manifold: TempContactManifold;
+  flipped: boolean;
 }
 
+export type CollisionExitPayload = {
+  target: RapierRigidBody;
+  collider: RapierCollider;
+}
+
+export type CollisionEnterCallback = (payload: CollisionEnterPayload) => void;
```

Comment:
Do you prefer `Callback` or `Handler`? I've come to prefer `handler`, but I know I've used `callback` in other trivial places. Seeing it like this I feel like changing it.

Reply:
`Callback` was just what I was used to, and I'd be happy to change it for consistency's sake. Which one would you prefer to go with?

Reply:
I think `handler`! Will change other occasions as well.

Reply:
I've renamed it to Handler.